// Copyright 2019 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.
//
// Sync protocol datatype extension for security events.

// If you change or add any fields in this file, update proto_visitors.h and
// potentially proto_enum_conversions.{h, cc}.

syntax = "proto2";

option java_multiple_files = true;
option java_package = "org.chromium.components.sync.protocol";

option optimize_for = LITE_RUNTIME;

package sync_pb;

import "components/sync/protocol/gaia_password_reuse.proto";

message SecurityEventSpecifics {
  // The specific security event to record.
  oneof event { GaiaPasswordReuse gaia_password_reuse_event = 1; }

  // Time of event, as measured by client in microseconds since Windows epoch.
  optional int64 event_time_usec = 2;
}
